code/game/objects/items/tools/shovel_tools.dm

```diff
@@ -67,6 +67,9 @@
 		return
 
 	if(isturf(target))
+		if (sharp)
```

Comment:
Put this where the shovel gets sharpened. Since if its here it will set the var to 10 everytime the shovel starts to dig.

Reply:
Done. Is this fine?

Reply:
Thats perfect!